Sources/UI/Views/AvatarView.swift

```diff
@@ -14,87 +14,130 @@ import Nuke
 public final class AvatarView: EscapeBridgingImageView<Void>, Reusable {
     
     private var imageTask: ImageTask?
+    private var style = AvatarViewStyle()
     
-    private lazy var avatarLabel: UILabel = {
+    public private(set) lazy var avatarLabel: UILabel = {
         let label = UILabel(frame: .zero)
         label.textAlignment = .center
         label.isHidden = true
-        label.preferredMaxLayoutWidth = 2 * layer.cornerRadius
+        label.preferredMaxLayoutWidth = 2 * style.radius
         return label
     }()
     
     /// Create a AvatarView with a given corner radius.
```

Comment:
Needs updating

Reply:
What do you mean?

Reply:
The initializer changed, no longer accepts a corner radius

Reply:
I mean the documentation
> /// Create a AvatarView with a given corner radius.

needs updating to
> /// Create an AvatarView with a given AvatarViewStyle

Reply:
Fixed.